### DISCUSSION TITLE
Make it work without Filament?

### DISCUSSION THREAD

**Discussion by andreas-freitag** [awcodes/filament-tiptap-editor#499] [General]
Hey, I know it's a long shot, but I was wondering if you had any plans for releasing the tiptap editor as a standalone blade component (for use with or without livewire) without the need for filament.

I would really like to use it but I don't want to go the filament way.

**Comment by awcodes** [comment]
I have no plans to do this. It's too entangled with filament, alpine and livewire. It would be possible without filament, but at the cost of loosing the ability to support custom blocks.

**Reply by andreas-freitag** [reply]
Sorry to hear that, but understandable. Thanks for your reply!

**Comment by awcodes** [comment]
I think Caleb might be working on something for Flux, not 100% sure though.
